files/preStudies/preStudies.tex

```diff
@@ -2,6 +2,8 @@ \chapter{Pre-study}
 
 Here follows pre-studies on technologies and tools to be used throughout this project, conducted in the initial phase of the project.
 
+When researching technologies, focus will be on new cutting edge technologies. Using new technology comes with a risk as they are not as battle tested as older technologies. They often have more bugs and this can cause problems during development but they also often provide better performance and new features or advantages over their counterpart. We think the trade of is worth a great deal. We get experience with new technologies that could be relevant for years to come as well as build a product with a enhanced features and performance.
```

Comment:
Last sentence: remove the 'a' -> 'product with enhanced features and performance'.